src/util.cpp

```diff
@@ -1446,11 +1448,20 @@ string FormatFullVersion()
 
 #endif
 
+double Round(double d, int place)
+{
+    const double accuracy = place != 0
+            ? std::pow(10, place)
+            : 1;
```

Comment:
>  pow(base, ±0) returns 1 for any base, even when base is NaN 

So no need for that check for pace being zero. Interesting things happen when negative is passed.
Some may say, it rounds 6.022 to -1 places as expected to 10. If that is not desired, i propose the following modification.

```
double accuracy = place > 0
            ? std::pow(10, place)
            : 1;
```
Btw: avoid negation in condition of ternary operator.